configs/development.json

```diff
@@ -414,7 +414,8 @@
   "agent": {
     "rinkeby": {
       "rpcStyle": "ethereum",
-      "db": "/usr/share/nomad",
+      "db": "db_path",
```

Comment:
Permission denied for creating folders in `/usr/share`

Reply:
cc @yourbuddyconner

Reply:
@yourbuddyconner wdya want the db path to be?

Reply:
lets do `./db`